deployments/container/Dockerfile.ubi9

```diff
@@ -16,20 +16,21 @@ FROM nvcr.io/nvidia/cuda:12.9.1-devel-ubi9 AS builder
 
 ARG CUDA_VERSION
 RUN dnf install -y --allowerasing  \
-        curl \
+        curl cmake \
```

Comment:
Out of curiosity, why do we need this dependency?

Reply:
See my commit message. Starting with v12.8 of https://github.com/NVIDIA/cuda-samples the build procedure has changed.